### PR TITLE
NEWS: add v3.2.8

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,6 @@
+etcd v3.2.8 (2017-09-11)
+- fix v2 client failover to next endpoint on oneshot failure
```
Comment (heyitsanthony) [inline NEWS]: s/oneshot/mutable operation/

**Comment by gyuho** [comment]
Merging this afternoon, after publishing release.

**Comment by xiang90** [comment]
@gyuho shall we get this merged?

**Comment by gyuho** [comment]
I will merge right before the release this Friday.

**Pull request merged**